src/routes/data-router.ts

```diff
@@ -0,0 +1,69 @@
+import { Router } from "express";
+import { getRepository } from "typeorm";
+import { Todo } from "../entities/todo";
+
+const dataRouter = Router();
+
+dataRouter.get("/Tasks", async (req, res) => {
+  try {
+    const todoRepository = getRepository(Todo);
+    const todos = await todoRepository.find({ order: { id: "ASC" } });
```

Comment:
לא חייב לשים את ה repository במשתנה בפני עצמו, בצוות אנחנו בד"כ לא עושים את זה
```suggestion
    const todos = await getRepository(Todo).find({ order: { id: "ASC" } });
```

Reply:
טופל